test/ecto/migrator_test.exs

```diff
@@ -466,7 +466,7 @@ defmodule Ecto.MigratorTest do
     end
 
     test "version migrations stop before all have been run" do
-      assert run(TestRepo, [{13, ChangeMigration}, {14, UpDownMigration}], :up, to: 13, log: false) == [13]
+      assert run(TestRepo, [{13, ChangeMigration}, {14, UpDownMigration}], :up, to: 13, log: false, repo_name: :tenant_db) == [13]
```

Comment:
Instead of changing an existing test, let's add a separate test that asserts on multiple migrations using the tenant_db and making sure it does to the tenant_db repo!

Reply:
I've added `test/ecto/tenant_migrator_test.exs` but I wonder if it isn't too much redundant, actually I've copy/pasted code just to test this one option, if this is ok I'll update documentation

Reply:
Right, I think a whole migrator file is too much, we just need some basic smoke tests. Sorry for not being clear. Can you please keep a merge a minimal amount of tests into the existing migrator suite? Then we just need the docs and we should be ready for merging.

Reply:
indeed it was unnecessary, I've added smoke tests and `repo_name` option documentation. Let me know if there's anything else needed :)